common/setups/rasr/rasr_system.py

```diff
@@ -102,6 +114,14 @@ def _init_corpus(self, corpus_key: str):
             concurrent=self.concurrent[corpus_key],
             segment_path=segm_corpus_job.out_segment_path,
         )
+
+        if self.rasr_binary_path is not None and self.rasr_arch is not None:
+            self.crp[corpus_key].set_executables(
+                rasr_binary_path=self.rasr_binary_path, rasr_arch=self.rasr_arch
+            )
+        elif self.rasr_binary_path is not None and self.rasr_arch is None:
+            self.crp[corpus_key].set_executables(rasr_binary_path=self.rasr_binary_path)
+
```

Comment:
```suggestion
        self.crp[corpus_key].set_executables(
            rasr_binary_path=self.rasr_binary_path, rasr_arch=self.rasr_arch
        )
```
If the defaults are not None any more, we don't need to test for it.